lib/presentation/bloc/button/login_cubit.dart

```diff
@@ -0,0 +1,43 @@
+// ignore_for_file: depend_on_referenced_packages
+
+import 'package:bloc/bloc.dart';
+import 'package:injectable/injectable.dart';
+import 'package:truckbill/domain/auth/repository/auth_repository.dart';
+import 'package:truckbill/presentation/bloc/button/login_state.dart';
+
+@injectable
+class LoginCubit extends Cubit<LoginState> {
```

Comment:
tutaj żeby używać akcji robisz
```suggestion
class LoginCubit extends ActionCubit<LoginState,LoginAction> {
```